lib/index.js

```diff
@@ -140,57 +154,67 @@ Intercom.prototype.track = function(event, fn){
 };
 
 /**
- * Make a request to add to an existing Bulk Event Job and then unlock
+ * Make a request to add to an existing Bulk Job and then unlock
+ *
+ * https://developers.intercom.com/reference#adding-to-a-bulk-job
  *
  * @api private
  * @param {String} jobId
  * @param {String} key
- * @param {Object} track
+ * @param {Facade} msg
  * @param {Function} fn
  */
 
-Intercom.prototype.addToExisting = function(jobId, track, key, fn){
+Intercom.prototype.addToExisting = function(jobId, msg, key, fn){
   var self = this;
-  var data = mapper.track(track);
+  var type = msg.action();
+  var data = mapper[type](msg);
   data.job = { id: jobId };
+  var endpoint = '/bulk';
+  type === 'track' ? endpoint += '/events' : endpoint += '/users';
```

Comment:
This is much more readable as `endpoint += (type === track) ? '/events' : '/users';`

It's probably better to just use the `url-join` package and do
`var endpoint = urlJoin('/bulk', (type === 'track') ? 'events' : 'users')`